EasyClangComplete.sublime-settings

```diff
@@ -231,25 +231,12 @@
   // quality of the completions, however, in some corner cases it might cause
   // completions to fail entirely.
   "target_compilers": {
-    "C":              null,
+    "C":              {"compiler": "clang", "flags": ["-flag1", "-flag2"]},
```

Comment:
These should be some arguments with which the plugin should be working by default. Do we need the `flags` entry here? Should we leave the value at `null` and add the appropriate information in the settings instead?